### PR TITLE
chore(profileInfo) Hidden follow button on a user own profile

### LINKED ISSUE #539: Remove follow button on a user's own profile
**Is your feature request related to a problem? Please describe.**
When clicking on my own profile, I don't expect to see a follow button when I'm logged in with the same account or the same account is selected. see screenshot.

**Describe the solution you'd like**
When I view the same account that is currently active, there should be no follow / unfollow button.

image

### PR DESCRIPTION
**Pull request by developerfred**
close #539

### CODE REVIEW

File: src/common/components/ProfileInfoContent.tsx [hunk 1]
```diff
@@ -22,9 +23,20 @@ const ProfileInfoContent: React.FC<ProfileInfoContentProps> = ({
   hideBio = false,
   wideFormat = false,
 }) => {
+  const currentUserFid = useDataStore((state) => {
+    const username = window?.localStorage.getItem('active_account');
```
Comment (hellno) [inline src/common/components/ProfileInfoContent.tsx:27]: the active account is not stored in localStorage.
if an account is selected, it is stored in:
`AccountStore` which has `accounts` and `selectedAccountIdx`.
a common pattern to get it is: 
`const userFid = useAccountStore((state) => state.accounts[state.selectedAccountIdx]?.platformAccountId);`

**Review by hellno — CHANGES_REQUESTED** [review]
thanks for opening this! 🙏
a small change is needed to make this work with how we handle accounts and their states in herocast

File: src/common/components/ProfileInfoContent.tsx (hunk 1 above)
Comment (developerfred) [reply src/common/components/ProfileInfoContent.tsx:27]: Thanks for quikly response, I will make change now.

**Review by hellno — APPROVED** [review]
🔥🔥🔥

**Pull request merged**